Frontend/src/api/CollaborationRequests.jsx

```
@@ -44,7 +51,15 @@ const CollaborationRequests = () => {
     content = (
       <ul className="space-y-4">
         {requests.map((req) => (
-          <RequestCard key={req.id} req={req} />
+          <div key={req.id} className="flex justify-between items-center p-4 border rounded-lg">
+            <RequestCard req={req} />
+            <button
+              className="bg-green-500 text-white px-4 py-2 rounded hover:bg-green-600"
+              onClick={() => handleAccept(req.id)}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)